xchange-stream-coinmate/src/test/java/info/bitrich/xchangestream/coinmate/v2/CoinmateManualExample.java

```diff
@@ -11,17 +15,26 @@
 
   public static void main(String[] args) {
 
+    ExchangeSpecification exSpec = new CoinmateStreamingExchange().getDefaultExchangeSpecification();
+    exSpec.setUserName("2985");
+    exSpec.setApiKey("WRBBKkKPp43ST5yeUhfsba1E2ZikE8afQDSxDLCKSXg");
+    exSpec.setSecretKey("wyhudaS7irE3MXTRR8F9FSxt6QKo0-Wn8GQSvgLiCXk");
     StreamingExchange exchange =
-        StreamingExchangeFactory.INSTANCE.createExchange(CoinmateStreamingExchange.class);
+        StreamingExchangeFactory.INSTANCE.createExchange(exSpec);
     exchange.connect().blockingAwait();
 
-    exchange
-        .getStreamingMarketDataService()
-        .getOrderBook(CurrencyPair.BTC_EUR)
-        .subscribe(
-            orderBook -> {
-              LOG.info("Ask: {}", orderBook.getAsks().get(0));
-              LOG.info("Bid: {}", orderBook.getBids().get(0));
+//    exchange
```

Comment:
can you get rid of this comment?

Reply:
Thanks, also removed credentials

Reply:
Thanks, also removed credentials

Reply:
You should delete those api keys in your app so as to not compromise yourself

Reply:
Thanks for noticing! I have disabled those API keys